src/vm/eventpipe.h

```diff
@@ -51,6 +51,7 @@ class EventPipe
         uint32_t numProviders,
         EventPipeSessionType sessionType,
         EventPipeSerializationFormat format,
+        bool rundownRequested,
```

Comment:
missing `const`

Reply:
Most code in the runtime does not specify const on arguments that are passed by value. My personal opinion is that it makes code more verbose while adding little value and differs from most of the other code in the runtime, but if the concensus on the team is that we should be doing it I have no strong objection. I would suggest that if we do want to hold a certain style in the EventPipe code then a good starting point is to make the existing code follow the pattern. This lets developers match the existing style rather than trying to remember different sets of rules for different files.

Reply:
I should have explained myself here: Declaration and definition are different by the `const` keyword.

Reply:
https://github.com/dotnet/coreclr/pull/25495/files#diff-0e879bae13329f135aa0ac1641eef454R171

Reply:
ah my mistake, I missed what you after there, sorry for the noise